Aio.Multidataset.Rest.Connector/DatasetSamplerFactory.cs

```diff
@@ -1,46 +1,51 @@
-namespace Aio.Multidataset.Rest.Connector;
+// Copyright (c) Microsoft Corporation.
```

Comment:
This is not present consistently across the files, we may either drop it or add everywhere?

Reply:
@tompaana ?